storage/pin/pin.go

```diff
@@ -241,13 +241,13 @@ func (p *API) UnpinFiles(addr []byte, credentials string) error {
 //     3) the number of times that particular file or collection is pinned.
 func (p *API) ListPins() ([]PinInfo, error) {
 	pinnedFiles := make([]PinInfo, 0)
-	iterFunc := func(key []byte, value []byte) {
+	iterFunc := func(key []byte, value []byte) (stop bool, err error) {
 		hash := string(key[4:])
 		pinInfo := PinInfo{}
-		err := pinInfo.UnmarshalBinary(value)
+		err = pinInfo.UnmarshalBinary(value)
```

Comment:
are you pulling changes from `master` or are you rebasing...? this seems misplaced....

Reply:
Why do you think so? I rebased on latest master, but the case is also that our PR changes the interface of the `StateStore`, which requires this to be changed here. We have synced with @jmozah about this and it all seems ok. 

What is your objection here?

Reply:
because this is related to content pinning and this PR is the SWAP PR... and where do you see an objection? :question:

Reply:
i believe @holisticode referred to this comment:
> this seems misplaced....

when he talked about an "objection".

in any case, i don't see an issue here, so let's consider this resolved unless @acud has anything else to add here.